float64 longitude
float64 latitude
float64 altitude
---
int8 val
